<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="count_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>31</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
	font: 12pt &quot;Osaka&quot;;
	background-color : rgb(210, 195, 255);
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sum_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box_time">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_time">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>лет</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_box_prc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>175</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_per_mon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>180</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_over">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>126</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_total">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_button_ann">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Аннуитетные</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_button_diff">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Osaka</family>
    </font>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
